types/backbone/index.d.ts

```diff
@@ -310,12 +310,12 @@ declare namespace Backbone {
         values(): any[];
         pairs(): any[];
         invert(): any;
-        pick(keys: Array<keyof T & string>): any;
-        pick(...keys: Array<keyof T & string>): any;
-        pick(fn: (value: any, key: any, object: any) => any): any;
-        omit(keys: Array<keyof T & string>): any;
-        omit(...keys: Array<keyof T & string>): any;
-        omit(fn: (value: any, key: any, object: any) => any): any;
+        pick<a extends keyof T & string>(keys: a[]): Partial<_Pick<T, a>>;
+        pick<a extends keyof T & string>(...keys: a[]): Partial<_Pick<T, a>>;
+        pick(fn: (value: any, key: any, object: any) => any): Partial<T>;
+        omit<a extends keyof T & string>(keys: a[]): Partial<_Omit<T, a>>;
+        omit<a extends keyof T & string>(...keys: a[]): Partial<_Omit<T, a>>;
```

Comment:
Also, the type `keyof T & string` appears so often that I wonder whether it warrants a dedicated helper type.

Reply:
It would be great if you could tell me how to do it.

Reply:
I would think something like

```ts
type StringKey<T> = keyof T & string;
```

and then

```ts
pick<A extends StringKey<T>>(keys: A[]): Partial<Pick<T, A>>;
```

and so forth.